packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -28,6 +30,27 @@ const StyledPlayButton = styled(PlayButton)`
   /* stylelint-enable */
 `;
 
+const NoScriptHiddenStyle = (
```

Comment:
Instead of manually injecting CSS outside of Styled Components, have you considered a more Styled Components oriented approach?

If we had a `.no-js` class on the Placeholder, which was removed by default, we could use that as a hook when JS isn't present. This would work in a Styled Component in the following way:

```
const PlayButton = styled.button`
  .no-js & {
    display: none;
  }
`;
```

Reply:
this would be tricky to achieve in simorgh when tracking the js disabled mode because the initial render would always be done by the server and when js is enabled, initial render will have the `.no-js` styles

Reply:
How bad would this look if we don't attempt to apply no-JS-specific styles? @rhenshaw56 could you post some screenshots?

I think we're in danger of making the solution too complicated in order to precisely match the UX, whereas the primary goal is to have a legible (and accessible) explanation. Finessing the appearance could be done later if needed.

Reply:
> How bad would this look if we don't attempt to apply no-JS-specific styles?

@jamesdonoh we would have the play button overlay the no-js message.
something like

![Screen Shot 2019-11-21 at 17 38 52](https://user-images.githubusercontent.com/25608386/69357710-df9ccb80-0c85-11ea-812b-551b11f63c42.png)

Reply:
OK, so I see the error message is not legible. Is there some way we could make the message display in front of the play button without using non-JS-specific styles?

Reply:
@jamesdonoh we could but the behavior might look weird as hovering on the placeholder changes the color of the button to red (so part of the no-js text background will be red), coupled with the fact that the video is not present, the play button screen reader text would always read out

Reply:
have you seen my suggestion [above](https://github.com/bbc/psammead/pull/2662#discussion_r349005026) ?

Reply:
@jamesdonoh @simonsinclair I have made the change using the method recommended on slack, here's a preview

![Screen Shot 2019-11-25 at 10 16 15](https://user-images.githubusercontent.com/25608386/69527597-f0db2600-0f6c-11ea-898e-3422f13beae1.png)